src/java/edu/slu/tpen/transfer/JsonLDExporter.java

```diff
@@ -50,16 +51,43 @@
  */
 public class JsonLDExporter {
 
+   
    /**
     * Holds data which will be serialised to JSON.
     */
    Map<String, Object> manifestData;
-
+public JsonLDExporter(Project proj, User u, String profile) throws SQLException, IOException
+{   
+   
+       Folio[] folios = proj.getFolios();
+       int projID = proj.getProjectID();
+       if (profile.contains("3")){       
+               System.out.println("This is Presi 3!");
+               manifestData = new LinkedHashMap<>();
+               String projName = getRbTok("SERVERURL") + "manifest/"+projID;
+               
+               manifestData.put("@context", "http://www.shared-canvas.org/ns/context.json");
```

Comment:
This is not the right value for "@context".  See the presentation 3 API Manifest example.